ietf/doc/views_conflict_review.py

```diff
@@ -392,10 +400,52 @@ def __init__(self, *args, **kwargs):
 
 @role_required("Secretariat","IRTF Chair","ISE")
 def start_review(request, name):
-    if has_role(request.user,"Secretariat"):
-        return start_review_as_secretariat(request,name)
+    '''Start the conflict review process.
+
+    Start the conflict review process, setting the initial 
+    shepherding AD, and possibly putting the review on a telechat.'''
+
+    def cleaned_notify(notify, doc):
+        if doc.stream.slug not in ['irtf', 'ise']:
+            return notify
+        stream_managers = [
+            re.sub('([^<]*<)?(.*)(>.*)', '\2', r)
+            for r in Recipient(slug='stream_managers').gather(streams=[doc.stream.slug])
+        ]
+        notifications = notify.split(',')
+        for mgr in stream_managers:
+            goods = [n for n in notifications if mgr not in n]
+            notifications = goods
```

Comment:
This has to be a bug? You are calculating and throwing away a value for all but the last iteration of a for loop, and then using the value of the last iteration?

Reply:
This code does set subtraction of the items in stream_managers from the items in notifications.  It will:
- Iterate over item `mgr` in `stream_managers`
  - Determine every item in `notifications` that does not match `mgr`, and store that in `goods`
  - Copy `goods` over to `notifications`

Each pass through 417-418 ensures that one item out of the `stream_managers` list is not present in the `notifications` list.

The `goods` variable only exists here because I'm uncertain how well the language handles iterating over a variable as it mutates.  Assigning it to a temporary variable removes that ambiguity.